pkg/ddc/jindofsx/transform.go

```diff
@@ -365,6 +365,17 @@ func (e *JindoFSxEngine) transformResources(runtime *datav1alpha1.JindoRuntime,
 		}
 	}
 
+	// mem set request
+	if e.getTieredStoreType(runtime) == 0 {
+		quotaString := strings.TrimRight(userQuotas, "g")
```

Comment:
What if `userQuotas = "1g,2g`? After a `TrimRight(userQuotas, "g")`, `quotaString` will be "1g,2" which cannot be Atoi?

Reply:
This change will only happen when tirestore is set to MEM, MEM does not support "2g, 3g", only "2g", a single quota